### ISSUE TITLE
confuse about weight printed in model.save

### ISSUE THREAD

**Issue by tengteng95** [RoyalVane/MMAN#2]
Hi, i wonder what's the meaning of 'weight L1' , 'weight L1 Attri' etc. printed in the function model:save() before saving the model state_dicts. It seems it has no relation with the lr scheduler, so I'm a little confused.

By the way, i want to know how do you train the baseline model with no discriminators? To be Specific, should the L1 loss be multiplied by lambda_A or not when no GAN loss applied?

**Comment by RoyalVane** [comment]
Yes, you are right. The print out results have been used in our debug version to test different lrs, but it haven't rightly modified in release version. Nevertheless,  it does not affect the final result. Besides, in our experiment, the L1 loss is still multiplied by lambda_A when no GAN loss is applied.
